flamedisx/xenon/x1t_sr1.py

```diff
@@ -214,13 +219,29 @@ class SR1Source:
     # FDC map
     path_drift_field_distortion_correction = 'nt_maps/XnT_3D_FDC_xyt_MLP_v0.2_B2d75n_C2d75n_G0d3p_A4d9p_T0d9n_PMTs1d3n_FSR0d65p.json'
 
+    # Drift velocity map
+    default_drift_velocity = DEFAULT_DRIFT_VELOCITY
+    variable_drift_velocity= True
+    path_drift_velocity='nt_maps/XnT_drift_velocity_map_r_z.json'
+
+    # SE gain map    
+    variable_se_gain= True
+    path_se_gain_map='nt_maps/XENONnT_se_xy_map_v1_mlp.json'    
```

Comment:
@eangelino could you link the internal xenon note documenting how this SE gain map was derived and how it should be used/interpreted?

Reply:
Also, how does this affect the final interpretation of g2? The information from `se_gain_relative` goes into the `electron_gain_mean` model function but we'll still compute the overall g2 from just the bestfit value for `single_electron_gain`?

Reply:
I am currently looking for a reference note. The map itself contains the the se_gain absolute values in PE for (x,y). Since we would like to keep the single electron gain as a parameter for the fit, the map is normalized by the average value. So, what is interpolated is the relative variation from the average value. This is added in order to consider dishomogeneities even leaving the se_gain average value free.

Reply:
Reference note for se_gain_map: 
https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenonnt:jianyang:wfsim_data_se_matching